src/deploy/osp_deployer/settings/config.py

```diff
@@ -459,106 +495,106 @@ def __init__(self, settings_file):
-                except AttributeError:
-                    node.is_controller = False
-                    pass
-                try:
-                    if node.is_compute == "true":
-                        node.is_compute = True
-                        self.compute_nodes.append(node)
-                except AttributeError:
-                    node.is_compute = False
-                    pass
-                try:
-                    if node.is_ceph_storage == "true":
-                        self.ceph_nodes.append(node)
-                        node.is_storage = True
-                except AttributeError:
-                    node.is_storage = False
-                    pass
-                try:
-                    if node.is_switch == "true":
-                        self.switches.append(node)
-                except AttributeError:
-                    self.nodes.append(node)
-                    pass
-                try:
-                    if node.skip_raid_config == "true":
-                        node.skip_raid_config = True
-                except AttributeError:
-                    node.skip_raid_config = False
-                    pass
-
-        Settings.settings = self
-
-    def get_curated_nics_settings(self):
-        nics_settings = self.get_nics_settings()
-        if 'sah_bond_opts' in nics_settings:
-            del nics_settings['sah_bond_opts']
-        if 'nic_env_file' in nics_settings:
-            del nics_settings['nic_env_file']
-        return nics_settings
-
-    def get_nics_settings(self):
-        return self.get_settings_section("Nodes Nics and Bonding Settings")
-
-    def get_settings_section(self, section):
-        dictr = {}
-        options = self.conf.options(section)
-        for option in options:
-            try:
-                dictr[option] = self.conf.get(section, option)
-                if dictr[option] == -1:
-                    logger.debug("skip: %s" % option)
-            except ConfigParser.NoSectionError:
-                logger.debug("exception on %s!" % option)
-                dictr[option] = None
-        return dictr
-
-    def get_version_info(self):
-        # Grab the source version info from either built .tar or git
-        try:
-            repo_release_txt = self.cloud_repo_dir + "/release.txt"
-            if os.path.isfile(repo_release_txt):
-                re_ = open(repo_release_txt, 'r').read()
-            else:
-                cmd = "cd " + self.cloud_repo_dir + ";" + \
-                      "git log | grep -m 1 'commit'"
-                re_ = subprocess.check_output(cmd,
-                                              stderr=subprocess.STDOUT,
-                                              shell=True).rstrip()
-            self.source_version = re_
-        except:
-            logger.debug("unconventional setup...can t" +
-                         " pick source version info")
-            self.source_version = "????"
```

Comment:
Not sure what happened here, but can you re apply changes to that file ? should only see changes here, yet most of the file is showing as different.

Reply:
Resolved in commit 0587d13.